ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/CommitContext.scala

```diff
@@ -23,7 +23,7 @@ import scala.collection.mutable
 private[kvutils] trait CommitContext {
```

Comment:
It's really weird that this is a trait. Perhaps it should be a class and accept `inputs`, `recordTime`, and `participantId` as parameters?

Definitely something for a subsequent changeset, though.

Reply:
Yes, it is -- we can fix that in a later PR.